app/mailers/report_mailer.rb

```diff
@@ -1,30 +1,32 @@
 class ReportMailer < ActionMailer::Base
-  default :from => AppConfig.mail.sender_address
+  default from: AppConfig.mail.sender_address
 
-  def new_report(type, id)
+  def self.new_report(type, id)
```

Comment:
Not applicable here, `#new_report` and `::new_report` are intentionally different methods.